### DISCUSSION TITLE
Lua script for telemetry.

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#4944] [Q&A]
I've been trying to build a Lua script that shows telemetry data (I know this can be viewed elsewhere, but I want to add some calculations and stuff eventually) and I can't get it to pull the data correctly. 

Anyway, my real question is, how exactly should the sensor ID be inputted? If it's showing in the settings as "0003   1".

 In the Lua script, should I input the ID as "0003"? "0003_1"? "0003 1"? Or is there another way to add sensor ID? I am using FlySky IA10B with the voltage collector module.

Help is appreciated!

**Comment by jrwieland** [comment]
There are existing widgets in the sd card you can use for reference
Here are a Couple other useful resources to look at when writing your script

Edge lua reference guide

https://luadoc.edgetx.org/

Also here is one that I use for both playing music while flying and receiving telemetry data

https://github.com/jrwieland/TaraniTunes-v4.x/tree/master/Color%20Screen%20Widget%2FEdgeTX

Send me a Dm and I can assist in whatever capacity you need
